build.c

```diff
@@ -555,8 +603,10 @@ build(void)
 		}
 		if (numjobs == 0)
 			break;
-		if (poll(fds, jobslen, -1) < 0)
-			fatal("poll:");
+		while (poll(fds, jobslen, -1) < 0) {
+			if (errno != EINTR)
+				fatal("poll:");
+		}
```

Comment:
Does blocking `SIGWINCH` prevent the need for checking for `EINTR` here?

Reply:
Actually, I think that since I removed the watching for SIGWINCH to rerun ioctl, and instead have it run before each status print, this change shouldn't necessary even without blocking SIGWINCH.